filters/filters.txt

```diff
@@ -4593,12 +4593,9 @@ themeslide.com##+js(nano-setInterval-booster.js, countdown, 1500)
 ! https://github.com/uBlockOrigin/uAssets/issues/1494
 ! https://github.com/NanoAdblocker/NanoFilters/issues/272
 ! https://github.com/NanoMeow/QuickReports/issues/25
-*$script,domain=aternos.org,3p,redirect-rule=noopjs
-@@||aternos.org^$generichide
-@@||amazon-adsystem.com/*/apstag.js$script,domain=aternos.org
-@@||snigelweb-com.videoplayerhub.com/videoloader.js$script,domain=aternos.org
-@@||static.h-bid.com^$css,script,domain=aternos.org
-@@*$xhr,3p,domain=aternos.org
+aternos.org##.ad
+aternos.org##+js(acis,$)
```

Comment:
Can this be made more specific by adding a second argument?
The second argument can be any string of the inline script you want to abort.
https://github.com/gorhill/uBlock/wiki/Resources-Library#acisjs-